### PR TITLE
Added Attribute details for key_vault_certificate.html.markdown

### LINKED ISSUE #9622: key_vault_certificate Attributes missing in Documentation
### Terraform (and AzureRM Provider) Version

Terraform v0.13.4

### Affected Resource(s)

* `azurerm_key_vault_certificate`
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_certificate

### Terraform Configuration Files

```hcl
key_vault_secret_id = data.azurerm_key_vault_certificate.mycertificate.secret_id
```

### Expected Behaviour

I expected `secret_id` to be listed in the Docs or a deprecation warning in Terraform, but neither is there. 
The data provider definitely exports it. I think the docs should reflect this.

### Actual Behaviour

`data.azurerm_key_vault_certificate.mycertificate.secret_id` exports a piece of data (`secret_id`) the docs don't list it having.

### Steps to Reproduce
Use `data.azurerm_key_vault_certificate.mycertificate.secret_id` in your config and watch it get the secret_id.

### References

* https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/key_vault_certificate
* https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_certificate

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### PR DESCRIPTION
**Pull request by DenWin**
Add details base on issue #9622

Fixes #9622

### CODE REVIEW

File: website/docs/d/key_vault_certificate.html.markdown [hunk 1]
```diff
@@ -35,18 +35,31 @@ output "certificate_thumbprint" {
...
+* `id` - The Key Vault Certificate ID.
+*
+* `name` - Specifies the name of the Key Vault Certificate.
+*
+* `secret_id` - The ID of the associated Key Vault Secret.
+*
+* `version` - The current version of the Key Vault Certificate.
+*
+* `certificate_data` - The raw Key Vault Certificate data represented as a hexadecimal string.
+*
```
Comment (tombuildsstuff) [inline website/docs/d/key_vault_certificate.html.markdown:60]: this is superfluous - so could we remove this in favour of a blank line?

```suggestion

```

File: website/docs/d/key_vault_certificate.html.markdown [hunk 2]
```diff
@@ -35,18 +35,31 @@ output "certificate_thumbprint" {
...
 
+
+* `id` - The Key Vault Certificate ID.
+*
+* `name` - Specifies the name of the Key Vault Certificate.
+*
+* `secret_id` - The ID of the associated Key Vault Secret.
+*
+* `version` - The current version of the Key Vault Certificate.
+*
```
Comment (tombuildsstuff) [inline website/docs/d/key_vault_certificate.html.markdown:58]: this is superfluous - so could we remove this in favour of a blank line?

```suggestion

```

File: website/docs/d/key_vault_certificate.html.markdown [hunk 3]
```diff
@@ -35,18 +35,31 @@ output "certificate_thumbprint" {
...
 
 The following attributes are exported:
 
+
+* `id` - The Key Vault Certificate ID.
+*
+* `name` - Specifies the name of the Key Vault Certificate.
+*
+* `secret_id` - The ID of the associated Key Vault Secret.
+*
```
Comment (tombuildsstuff) [inline website/docs/d/key_vault_certificate.html.markdown:56]: this is superfluous - so could we remove this in favour of a blank line?

```suggestion

```

File: website/docs/d/key_vault_certificate.html.markdown [hunk 4]
```diff
@@ -35,18 +35,31 @@ output "certificate_thumbprint" {
...
 
 ## Attributes Reference
 
 The following attributes are exported:
 
+
+* `id` - The Key Vault Certificate ID.
+*
+* `name` - Specifies the name of the Key Vault Certificate.
+*
```
Comment (tombuildsstuff) [inline website/docs/d/key_vault_certificate.html.markdown:54]: this is superfluous - so could we remove this in favour of a blank line?

```suggestion

```

File: website/docs/d/key_vault_certificate.html.markdown [hunk 5]
```diff
@@ -35,18 +35,31 @@ output "certificate_thumbprint" {
...
 
 **NOTE:** The vault must be in the same subscription as the provider. If the vault is in another subscription, you must create an aliased provider for that subscription.
 
 ## Attributes Reference
 
 The following attributes are exported:
 
+
+* `id` - The Key Vault Certificate ID.
+*
```
Comment (tombuildsstuff) [inline website/docs/d/key_vault_certificate.html.markdown:52]: this is superfluous - so could we remove this in favour of a blank line?

```suggestion

```

**Review by tombuildsstuff — APPROVED** [review]
hey @DenWin 

Thanks for this PR :)

Taking a look through this mostly LGTM - if we can remove the superflurous `*`'s in favour of a blank line to ensure this renders correctly then this otherwise LGTM 👍 

Thanks!

**Comment by DenWin** [comment]
I cloesed this PR in favor of #9685

**Pull request closed without merging**